### PR TITLE
Add flag to ignore signature

### PR DESCRIPTION
**Pull request by n0rad**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -485,18 +486,18 @@ func abortUpload(num int) error {
...
-
-	tmpsig := path.Join(directory, "tmp", strconv.Itoa(num)+".asc")
-	_, err = os.Stat(tmpsig)
-	if err == nil {
-		err = os.Remove(tmpsig)
-		if err != nil {
+	tmpsig := path.Join(directory, "tmp", strconv.Itoa(num) + ".asc")
+	if _, err := os.Stat(tmpsig); err == nil {
+		_, err = os.Stat(tmpsig)
+		if err == nil {
```
Comment (cgonyeo) [inline main.go]: This looks like you call `os.Stat` twice in a row on `tmpsig`, checking that err is nil both times. Is this necessary?

Comment (cgonyeo) [reply main.go]: You also don't seem to be using the new flag in this block. Are the changes here necessary?

File: main.go [hunk 2]
```diff
@@ -517,10 +518,12 @@ func finishUpload(num int) error {
...
-	err = os.Rename(path.Join(directory, "tmp", strconv.Itoa(num)+".asc"),
-		path.Join(directory, up.Image+".asc"))
-	if err != nil {
-		return err
+	tmpsig := path.Join(directory, "tmp", strconv.Itoa(num) + ".asc")
+	if _, err := os.Stat(tmpsig); err == nil {
+		err = os.Rename(tmpsig, path.Join(directory, up.Image+".asc"))
+		if err != nil {
+			return err
+		}
```
Comment (cgonyeo) [inline main.go:532]: Same thing here, the `unsigned` flag isn't checked anywhere here. Are the changes here necessary?

Comment (n0rad) [reply main.go:532]: its required or the upload will complete with the Rename error.

I think checking if the file exists is safer than looking for the flag. But I can change if you want.

File: main.go (hunk 1 above)
Comment (n0rad) [reply main.go]: Sorry copy paste too fast

**Comment by cgonyeo** [comment]
LGTM now. Thanks!

**Pull request merged**